chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -16,6 +17,10 @@
 #include "brave/browser/widevine/widevine_utils.h"
 #endif
 
+#if BUILDFLAG(CRYPTO_DOT_COM_ENABLED)
+#include "brave/components/crypto_dot_com/browser/crypto_dot_com_pref_utils.h"
```

Comment:
Please, make sure to add deps to //chrome/browser (via brave/browser/sources.gni) to `brave/components/crypto_dot_com/browser/buildflags` and `//brave/components/crypto_dot_com/browser`

Reply:
Addressed by https://github.com/brave/brave-core/pull/7699/commits/bb8c25c049f23b473d677789b9fd4dcf08a00ab5